### PR TITLE
Allow saving cookie for cross-origin requests when in CORS mode.

### PR DESCRIPTION
**Pull request by lauravuo**
* allow saving cookie when in CORS mode
* fix deprecation related to origin param

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -107,12 +107,14 @@ func main() {
 		jwt.SetJWTSecret(jwtSecret)
 	}
 
+	rpOrigins := strings.Split(rpOrigin, ",")
```
Comment (lainio) [inline main.go:110]: should the flag `rpOrigin` usage be updated with then new syntax?

**Review by lainio — APPROVED** [review]
One comment about possible missing CLI API syntax update.

**Comment by lauravuo** [comment]
It seems that this change breaks cookie logic for headless authenticator. Let's postpone for now as this was apparently needed only for webauthn stuff from chrome extension.

**Comment by lainio** [comment]
> It seems that this change breaks cookie logic for headless authenticator. Let's postpone for now as this was apparently needed only for webauthn stuff from chrome extension.

I dunno if it helps then, but for future knowledge: the headless authenticator code includes the capability to keep cookies over process startups. It was needed for ISVA endpoint tests. Maybe some sort of combination of 'everything' would be needed. Let's discuss this online.

**Pull request closed without merging**